dotnet/src/SemanticKernel/TemplateEngine/PromptTemplateEngine.cs

```diff
@@ -75,7 +80,10 @@ public async Task<string> RenderAsync(IList<Block> blocks, SKContext context)
                     break;
```

Comment:
```suggestion
                    tasks.Add(Task.FromResult<string>(staticBlock.Render(context.Variables)));
                    break;
```
(...and remove the result.Append here)